pde2e-image/lib/darwin/runner.sh

```
@@ -71,9 +79,39 @@ done
 # SCRIPT ENVIRONMENT INITIALIZATION
 ####################################################################
 
+if [ $debug == "1" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)